test/integration_test/lib/util/test_author.py

```diff
@@ -0,0 +1,12 @@
+from lib.util.author import *
+import json
+
+def test_get_uid_map():
+
+	input_data='./test/integration_test/data/clean_data.json'
+	test_map='./test/integration_test/data/author_uid_map_test.json'
+	expected_set=json.load(open('./test/integration_test/data/author_uid_map.json')).keys()
+	
+	assert get_uid_map(input_data,test_map).keys()==expected_set
+    assert json.load(open(test_map)).keys()==expected_set
```

Comment:
@tejalwakchoure The indentation for this line is wrong, so the tests are not running properly. Please see [travis log](https://travis-ci.org/DeveloperCAP/MLCAT/jobs/347684036#L636).

Reply:
@prasadtalasila I have made the changes, the test seems to be passing now in the travis log.